internal/cloud_connector/message_handlers.go

```diff
@@ -208,9 +207,9 @@ func isDuplicateOrOldMessage(currentConnectionState domain.ConnectorClientState,
 	return false
 }
 
-func processDispatchers(sourcesRecorder controller.SourcesRecorder, identity domain.Identity, account domain.AccountID, orgID domain.OrgID, clientId domain.ClientID, handshakePayload map[string]interface{}) {
+func processDispatchers(logger *logrus.Entry, sourcesRecorder controller.SourcesRecorder, identity domain.Identity, account domain.AccountID, orgID domain.OrgID, clientId domain.ClientID, handshakePayload map[string]interface{}) {
 
-	logger := logger.Log.WithFields(logrus.Fields{"client_id": clientId, "account": account, "org_id": orgID})
+	logger = logger.WithFields(logrus.Fields{"client_id": clientId, "account": account, "org_id": orgID})
```

Comment:
This line can be removed now as we are setting the client_id, account and org_id in the logger before this function is called.